packages/std/src/math/mod.rs

```diff
@@ -64,14 +72,31 @@ mod tests {
     {
     }
 
+    trait SignedImpl<'a>: IntImpl<'a> + Neg<Output = Self> {}
```

Comment:
Should we also add Not? Maybe in `trait IntImpl<'a>`

Reply:
Yes, we should, but it's not implemented for most of the Uints, so I left it out of this PR

Reply:
okay, good, let's do this separately in 1.4: https://github.com/CosmWasm/cosmwasm/issues/1754. `Not` is nothing a user ever asked for.